Readme.md

```diff
@@ -21,3 +18,15 @@ API in this example:
 * [PivotGridControl.Appearance.Lines](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridAppearancesBase.Lines) property
 * [PivotGridLocalizer](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraPivotGrid.Localization.PivotGridLocalizer) class
 
+## Files to Review
+
+* [Form1.cs](./CS/RemoveGridLinesExample/Form1.cs) (VB: [Form1.vb](./VB/RemoveGridLinesExample/Form1.vb))
+
+## Documentation
+
+[Custom Draw](https://docs.devexpress.com/WindowsForms/1817/controls-and-libraries/pivot-grid/appearance/custom-draw)
+
+## More Examples
+
+- [Pivot Grid for WinForms - Change the Field Value Header Background](https://github.com/DevExpress-Examples/winforms-pivot-change-the-field-value-header-appearance-backcolor)
+- [Pivot Grid for WinForms - Draw a Custom Element When a User Hovering Over Field Value with a Mouse](https://github.com/DevExpress-Examples/winforms-pivot-grid-draw-a-custom-element-on-mouse-hover)

```

Comment:
Can you update this topic title as follows (in this link and in the actual topic)?

`Pivot Grid for WinForms - Draw a Custom Element When a User Hovers Over a Field Value with a Mouse`

Reply:
```suggestion
- [Pivot Grid for WinForms - Draw a Custom Element When a User Hovers Over a Field Value with a Mouse](https://github.com/DevExpress-Examples/winforms-pivot-grid-draw-a-custom-element-on-mouse-hover)

```